catalog-info.yml

```diff
@@ -5,8 +5,16 @@ metadata:
   name: terraform-aws-refarch-opensearch
   description: Terraform module for setting up Opensearch.
```

Comment:
Terraform module for setting up Opensearch. --> Terraform module for setting up OpenSearch.